packages/cli-tool/src/commands/generate/index.ts

```diff
@@ -16,27 +16,43 @@ export default class Generate extends Command {
     description: 'application name',
   }]
 
+  static flags = {
+    'version-control': Flags.string({
+      char: 'c',
+      description: 'version control to use in the project',
+      options: ['Github', 'Gitlab'],
+    }),
+  }
+
   public async run(): Promise<void> {
-    const {args} = await this.parse(Generate)
+    const {args, flags} = await this.parse(Generate)
+    const appName = args.appName
 
-    await this.config.runHook('initialize', {appName: args.appName}).then((value : Hook.Result<ChildProcess>) => {
+    await this.config.runHook('initialize', {appName: appName}).then((value : Hook.Result<ChildProcess>) => {
       value.successes[0].result.on('exit', () => {
-        this.log(`Generating Nimble React app with the project name = ${args.appName}!`)
-
-        // Continue on prompt for add on
-        this.promptVersionControl(args.appName)
+        this.log(`Generating Nimble React app with the project name = ${appName}!`)
+
+        const versionControl = flags['version-control']
```

Comment:
@carryall should we use `versionControl` instead of `version-control` so that we can do:
`        const versionControl = flags.versionControl;` 
instead? :)